src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void draw();
```

Comment:
methods in interfaces are public abstract by default

Reply:
done, thank you